rust/ql/src/queries/security/CWE-798/HardcodedCryptographicValue.ql

```diff
@@ -0,0 +1,57 @@
+/**
+ * @name Hard-coded cryptographic value
+ * @description Using hardcoded keys, passwords, salts or initialization
```

Comment:
I think we need to decide on a consistent spelling of hard-coded/hardcoded.

Reply:
I agree with this, and I think there's a bit of both in our collection of queries over the years.

Reply:
As far as I can tell both are correct.  There appear to be more existing uses of "hardcoded", but it's a big mix and the query `@name`s are consistently "hard-coded".  So in order to not break that I've gone with the latter.